aten/src/ATen/core/Dict_inl.h

```diff
@@ -61,7 +107,7 @@ Dict<Key, Value>::Dict(c10::intrusive_ptr<detail::DictImpl>&& impl): impl_(std::
 template<class Key, class Value>
 Dict<Key, Value>& Dict<Key, Value>::operator=(Dict&& rhs) noexcept {
   impl_ = std::move(rhs.impl_);
-  rhs.impl_ = make_intrusive<detail::DictImpl>();
+  rhs.impl_ = make_intrusive<detail::DictImpl>(impl_->elementTypes);
```

Comment:
how does this code compile? dict_map_type missing?

Reply:
fixed